src/ipsis/course-merge/CourseMergeOfferingCollectionEntity.js

```diff
@@ -10,11 +10,11 @@ export class CourseMergeOfferingCollectionEntity extends BaseCollectionEntity {
 		if (!this._entity) {
 			return;
 		}
-		return this._entity.entities;
+		return allEntities ?? this._entity.entities;
```

Comment:
```suggestion
		return this.allEntities ?? this._entity.entities;
```

Reply:
Yes. Done. Also, how to mark a PR that it not for review yet? :)

Reply:
You can open a PR as a draft or wait to add reviewers until you're ready for it to be reviewed :)

Reply:
It was a draft! XD 
But yea, I requested the review, my bad. Now it's ready.

Reply:
🤣 Shows how good I am at paying attention 😄

Reply:
Don't forget to remove `[WIP]` from your PR title.